projects/ion/src/lib/link/link.component.ts

```diff
@@ -0,0 +1,51 @@
+import { CommonModule, NgClass } from '@angular/common';
+import {
+  Component,
+  EventEmitter,
+  Input,
+  OnChanges,
+  Output,
+  SimpleChanges,
+} from '@angular/core';
+
+import { IonIconComponent } from '../icon';
+import { IonLinkProps } from './types';
+
+@Component({
+  standalone: true,
+  imports: [CommonModule, IonIconComponent, NgClass],
+  selector: 'ion-link',
+  templateUrl: './link.component.html',
+  styleUrls: ['./link.component.scss'],
+})
+export class IonLinkComponent implements OnChanges {
+  @Input() label?: IonLinkProps['label'] = '';
+  @Input() icon?: IonLinkProps['icon'];
+  @Input() iconSide?: IonLinkProps['iconSide'] = 'right';
+  @Input() size?: IonLinkProps['size'] = 'sm';
+  @Input() bold?: IonLinkProps['bold'] = false;
+  @Input() disabled?: IonLinkProps['disabled'] = false;
+  @Input() target?: IonLinkProps['target'] = '_self';
+  @Input() link?: IonLinkProps['link'];
+  @Output() ionOnClick: IonLinkProps['ionOnClick'] = new EventEmitter<void>();
+
+  public iconSize = 16;
+
+  public onClick(): void {
+    if (this.disabled) {
+      return;
+    }
+    this.ionOnClick.emit();
+  }
+
+  public ngOnChanges(changes: SimpleChanges): void {
+    const sizeControls = {
+      sm: 16,
+      md: 24,
+    };
+
+    if (changes['size']) {
+      this.iconSize = sizeControls[this.size!];
```

Comment:
I think that if you remove the `?` on `size?` prop, you be able to remove the `!` from size here too.

Reply:
@allan-chagas-brisa alert me that this comment is incorrect, and in fact, it is. I think to remove the `!` you could add a validation to guarantee that this.size is not undefined.